src/sms-verification/scripts/services.js

```diff
@@ -0,0 +1,40 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('user_management.sms-verification');
+
+    module.factory('smsVerificationFactory', [
+        '$http',
+        'userManagementSmsVerificationConfig',
+        function ($http, userManagementSmsVerificationConfig) {
+            var apiRoot = userManagementSmsVerificationConfig.apiRoot();
+
+            var verify =  {
+                post: function (activationCode, phoneNumber) {
```

Comment:
Possibly this could accept an object with keys in case a project ever needed to override and add extra fields to the POST data?